### PR TITLE
Fixing app freeze related to MMBaseSingleLineInput

### PR DESCRIPTION
**Pull request by VitorVieiraZ**
Small fix in `MMBaseSingleLineInput.qml` to prevent app freeze in the first method of reproducing issue.

Fixes (Partially) #3538

### CODE REVIEW

File: app/qml/form/editors/MMFormGalleryEditor.qml [hunk 1]
```diff
@@ -92,6 +92,7 @@ MMPrivateComponents.MMBaseInput {
         MouseArea {
           anchors.fill: parent
           onClicked: {
+            root.forceActiveFocus() //prevent app freeze as mentioned in issue #3538
```
Comment (tomasMizera) [inline app/qml/form/editors/MMFormGalleryEditor.qml:95]: Isn't this happening also for the other relations editor?

Comment (tomasMizera) [reply app/qml/form/editors/MMFormGalleryEditor.qml:95]: Additional question, is the crash happening only when taking a photo from within the child feature or is it reproducible for simple forms with an image editor?

Comment (VitorVieiraZ) [reply app/qml/form/editors/MMFormGalleryEditor.qml:95]: Indeed, the issue was also occurring in the `MMFormRelationReferenceEditor`. In the [bug report](https://github.com/MerginMaps/mobile/issues/3538) I also included the steps to reproduce the issue and added the solution in this pull request. 
I couldn't reproduce the crash when taking photos in simple forms with an image editor.

**Review by tomasMizera — APPROVED** [review]

**Comment by jozef-budac** [comment]
I can't simulate this om my device. Can you verify this @VitorVieiraZ ? thanks

**Pull request merged**